blueman/plugins/manager/Info.py

```diff
@@ -83,9 +83,10 @@ def on_accel_activated(group, dialog, key, flags):
         ('UUIDs', format_uuids),
         ('Modalias', None),
         ('Adapter', None),
-        ('ManufacturerData', None),
-        ('ServiceData', None),
-        ('AdvertisingData', None)
+        # FIXME below 3 we need some sample data to decode and display properly
+        ('ManufacturerData', lambda x: str(x)),
```

Comment:
Just

```suggestion
        ('ManufacturerData', str),
```

?

Reply:
Oh boy :roll_eyes:, fixed and also updated the commit message as I mentioned in the other PR.